### ISSUE TITLE
Firebase Deploying guide in Readme file

### ISSUE THREAD

**Issue by ShanikaEdiriweera** [cjsheets/FCC-Voting-App#3]
It would be great if the steps to deploy in Firebase are given in the Readme file.

eg:- first build the app, then `firebase init` , `firebase deploy`

**Comment by SavithriNandadasa** [comment]
Yeah. It would be great if you can atleast put a screenshot of the firebase DB
